sklearn/discriminant_analysis.py

```diff
@@ -93,7 +120,14 @@ def _class_means(X, y):
     return means
 
 
-def _class_cov(X, y, priors, shrinkage=None):
+def _class_cov(
+        X,
+        y,
+        priors=None,
```

Comment:
Why are you changing the default here? We kinda prefer the private functions and methods not to have much default values for the parameters for us to specify them all explicitly, which kinda helps with preventing certain bugs.